lib/strings.md

```diff
@@ -5,20 +5,40 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is a String?
 
-Replace me with your response...
+A string, simply put, is a collection of special characters, letters, and/or numbers. Strings can be as a simple as a name to something longer like an entire sentence or paragraph.
+
 
 # What are some examples of information that would be Strings as opposed to some other data type?
 
-Replace me with your response...
+Since strings are unique in that they can contain a variety of data types like numbers, spaces, letters, and/or special characters, something like a street address (ex: "1132 E. 22nd St.") would be considered a string since it contains, letters, numbers, and special characters. 
+
 
 # What is one way, using Ruby, to retrieve the 6th character in a String like `"Ada Lovelace"`? How about the 8th character? What happens if you try to retrieve the value of the _99th_ character (Or any character that doesn't exist)?
 
-Replace me with your response...
+Steps to retrieve the 6th character  in "Ada Lovelace":
+1) string1 = "Ada Lovelace"
+2) string1[5..5]
```

Comment:
The command `string_name[x..n]` is typically used to return a range/selection. How might you simplify the command, since we are just returning one character?